src/data/utils.js

```diff
@@ -23,13 +23,13 @@ const getAllPossibleQueryParams = (locationURl = null) => {
   );
 };
 
-export const handleURLUpdationOnLoad = pageName => {
+export const handleURLUpdationOnLoad = formName => {
   const queryParam = getAllPossibleQueryParams();
   if (!Object.prototype.hasOwnProperty.call(queryParam, AUTH_MODE)
-    || queryParam?.[AUTH_MODE] !== pageName) {
-    const url = new URL(window.location.href || process.env.LMS_BASE_URL);
+    || queryParam?.[AUTH_MODE] !== formName) {
+    const url = new URL(window.location.href || process.env.MARKETING_SITE_BASE_URL);
```

Comment:
We can get `MARKETING_SITE_BASE_URL` using `getConfig()` otherwise we need to initialize it in prospectus